docs/supported-technologies.asciidoc

```diff
@@ -80,6 +80,8 @@ The Node.js agent will automatically instrument the following modules to give yo
 |https://www.npmjs.com/package/elasticsearch-js[elasticsearch-js] |>=8.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/graphql[graphql] |>=0.7.0 <15.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/handlebars[handlebars] |* |Will instrument compile and render calls
+|https://www.npmjs.com/package/jade[jade] |* |Will instrument compile and render calls
+|https://www.npmjs.com/package/pug[pug] |* |Will instrument compile and render calls
```

Comment:
```suggestion
|https://www.npmjs.com/package/pug[pug] |'0.1.0 || >=2.0.1' |Will instrument compile and render calls
```

Reply:
We don't exclude broken versions from the "supported-technologies" page. So here it's fine to list just `>=0.1.0` (in this case I'd guess v0.0.0 was just broken as well, so feel free to just list `*` if you like)